OWNERS_ALIASES

```diff
@@ -70,6 +70,7 @@ aliases:
     - oshoval
     - phoracek
     - qinqon
+    - orelmisan
```

Comment:
I think the list needs to be updated based on active reviewers.
Most of the members in this list are not actively reviewing network PRs when assigned as reviewers.

@orelmisan , do you intend to be more active in this area from now on?

Reply:
Thank you @AlonaKaplan for considering me as a potential reviewer for SIG network.
I'm currently not focused on that area, so I do agree with @EdDev.

Reply:
ok, removed the commit.